css/styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+  font-family: "Open Sans", sans-serif;
+}
+
+h1,
+h2,
+label {
+  font-family: "Proza Libre", sans-serif;
+}
+
+header {
+  background: #562b81;
+  width: 100%;
+}
+
+.logo h1 {
+  font-size: 1.25rem;
+  font-weight: 400;
+  margin: 0.75rem 0;
+}
+
+h2 {
+  font-size: 2rem;
+  margin: 0.75rem 0;
+}
+
+label {
+  display: block;
+  font-size: 1rem;
+  font-weight: 600;
+  margin-bottom: 0.5rem;
+}
+
+.txt-color-white {
+  color: white;
+}
+
+button {
+  background-color: #562b81;
+  border-radius: 4px;
+  border: 0;
+}
+
+.button-save {
+  text-transform: uppercase;
+  height: 2rem;
+  width: 4.5rem;
+}
+
+input,
+textarea {
+  width: 100%;
+  border: 1px solid #b6b6b6;
+  border-radius: 6px;
+  padding: 0.5rem;
+}
+
+input {
+  height: 2.5rem;
+}
+
+textarea {
+  resize: none;
+  height: 9rem;
+}
+
+input:focus,
+textarea:focus {
+  outline: 2px solid #562b81;
+}
+
+img.entry-thumbnail {
+  object-fit: cover;
+  width: 100%;
+  height: 250px;
+}
+
+@media screen and (min-width: 768px) {
+  img.entry-thumbnail {
+    height: 100%;
+  }
+}
```

Comment:
Make sure you always have media queries as the last thing in a CSS file. There should be no general CSS rules below a media query ever.